contracts/src/v0.8/dev/ocr2dr/OCR2DROracle.sol

```diff
@@ -7,27 +7,40 @@ import "../AuthorizedReceiver.sol";
 import "../../ConfirmedOwner.sol";
 
 /**
- * @title OCR2DR oracle contract (stub for now)
+ * @title OCR2DR oracle contract
  */
 contract OCR2DROracle is OCR2DROracleInterface, AuthorizedReceiver, ConfirmedOwner {
   event OracleRequest(bytes32 requestId, bytes data);
   event OracleResponse(bytes32 requestId);
 
+  error EmptyRequestData();
   error InvalidRequestID();
```

Comment:
About ID validation: With OCR, we can expect duplicate transactions with results. Do we want to have a separate error when a duplicate response is detected vs. a completely incorrect ID?

Reply:
We could potentially keep track of all past IDs to detect a duplicate, but this would require additional gas.. and the value is little - just a different error. I suspect we will get a strong push back on this from audit group.